packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -547,11 +545,12 @@ export class DatabaseCluster extends DatabaseClusterNew {
 
     return new secretsmanager.SecretRotation(this, id, {
       secret: this.secret,
-      automaticallyAfter,
       application: this.singleUserRotationApplication,
       vpc: this.vpc,
       vpcSubnets: this.vpcSubnets,
       target: this,
+      ...options,
+      excludeCharacters: options.excludeCharacters ?? DEFAULT_PASSWORD_EXCLUDE_CHARS,
```

Comment:
`options.excludeCharacters = ''` is a legitimate option. Does ?? respect the empty string? I googled it but didn't find a definition anywhere.

Reply:
Yes, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator . This will allow `excludeCharacters = ''`.

Reply:
Learned something! Thanks!!!

Reply:
Actually, this is a great call out @ahammond! I tried it, and it actually didn't work! Not because of `??` though - as it turned out, [this line in SecretsManger](https://github.com/aws/aws-cdk/blob/d95af009a8d59393d019620aa8878098ce0aed1d/packages/%40aws-cdk/aws-secretsmanager/lib/secret-rotation.ts#L241-L241) needed to be changed to `props.excludeCharacters !== undefined`. After that, passing an empty string to `addRotation*()` works as expected!